docs/usage/shoot_status.md

```diff
@@ -101,10 +101,16 @@ It is meant to make the user aware of potential problems that might occur due to
 
 **`CACertificateValiditiesAcceptable`**:
 
-This constraints indicates that there is at least one CA certificate which expires in less than `1y`.
+This constraint indicates that there is at least one CA certificate which expires in less than `1y`.
 It will not be added to the `.status.constraints` if there is no such CA certificate.
 However, if it's visible, then a [credentials rotation operation](shoot_credentials_rotation.md#certificate-authorities) should be considered.
 
+**`CRDsWithConversionWebhooksPresent`**:
+
+This constraint indicates that there is at least one CustomResourceDefinition in the cluster which has multiple stored versions and a conversion webhook configured. This could break the reconciliation flow of a `Shoot` cluster in some cases. See https://github.com/gardener/gardener/issues/7471 for more details.
```

Comment:
```suggestion
This constraint indicates that there is at least one `CustomResourceDefinition` in the cluster which has multiple stored versions and a conversion webhook configured. This could break the reconciliation flow of a `Shoot` cluster in some cases. See https://github.com/gardener/gardener/issues/7471 for more details.
```